src/System.Json/src/System.Json.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0;$(NetMinimum)</TargetFrameworks>
```

Comment:
```suggestion
    <TargetFrameworks>$(NetMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
```

Reply:
Aside: I wonder if we could write down our project style guidelines somewhere, or even codify them.